bedrock/products/templates/products/vpn/includes/macros.html

```diff
@@ -108,7 +108,11 @@ <h2 class="vpn-hero-sub-heading">{{ subheading }}</h2>
     </a>
 
     <p class="availability-copy">
+    {% if switch('vpn-wave-vi') %}
+      {{ ftl('vpn-shared-available-countries-v6', fallback='vpn-shared-available-countries-v6') }}
+    {% else %}
       {{ ftl('vpn-shared-available-countries-v5', fallback='vpn-shared-available-countries-v4') }}
```

Comment:
Could we maybe retire v4 at this point, and remove the FTL string? (same for occurrences below)

```suggestion
      {{ ftl('vpn-shared-available-countries-v5') }}
```